CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 11.0.1 (14.12.2020)
```

Comment:
It should be major update, because someone may depend on the Puppeteer's API.

I think we should add links to these major releases:

1. https://github.com/puppeteer/puppeteer/releases/tag/v2.0.0
2. https://github.com/puppeteer/puppeteer/releases/tag/v3.0.0
3. https://github.com/puppeteer/puppeteer/releases/tag/v4.0.0
4. https://github.com/puppeteer/puppeteer/releases/tag/v5.0.0

To make it easier for users to find breaking changes.